vscode-wpilib/package.json

```diff
@@ -462,4 +465,4 @@
         "wpilib-riolog": "^0.2.0",
         "xml2js": "^0.4.19"
     }
-}
+}
```

Comment:
The EOL at EOF needs to not be removed from this.

Reply:
It was removed by npm, fixed.